src/language-js/needs-parens.js

```diff
@@ -945,6 +945,31 @@ function needsParens(path, options) {
           }
         }
       }
+
+      if (
+        parent.type === "TaggedTemplateExpression" &&
+        (node.type === "OptionalMemberExpression" ||
+          node.type === "OptionalCallExpression")
+      ) {
+        return true;
+      }
+
+      if (
+        parent.type === "ChainExpression" &&
+        path.grandparent.type === "TaggedTemplateExpression"
+      ) {
+        let object = node;
+
+        do {
+          if (object?.optional === true) {
+            return true;
+          }
+
+          object =
+            object.type === "CallExpression" ? object.callee : object.object;
+        } while (object);
```

Comment:
```suggestion
        return true;
```

Reply:
I'm sorry. If there is no optional chaining, it is just member expression, not chain expression. I understood what you said. Thank you for your review.